src/raven.js

```diff
@@ -1882,6 +1894,32 @@ Raven.prototype = {
   },
 
   _makeRequest: function(opts) {
+    // Auth is intentionally sent as part of query string (NOT as custom HTTP header) to avoid preflight CORS requests
+    var url = opts.url + '?' + urlencode(opts.auth);
+
+    if (supportsFetch()) {
+      return _window
```

Comment:
@kamilogorek here is an issue - now, if you don't pass `{ credentials:"include" }` (include or same-origin) fetch won't pass cookies. Not passing cookies breaks authorization for hosted servers that requires it :(

Reply:
Hit this breakage in prod today.

Reply:
Yes, this.